easybuild/easyconfigs/r/rasterstats/rasterstats-0.19.0-foss-2022a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
+
+name = 'rasterstats'
+version = '0.19.0'
+
+homepage = 'https://github.com/perrygeo/python-rasterstats'
+description = "rasterstats is a Python module for summarizing geospatial raster datasets based on vector geometries."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('Shapely', '1.8.2'),
+    ('Fiona', '1.8.21'),
+    ('rasterio', '1.3.4'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('affine', '2.3.1', {
+        'checksums': ['d676de66157ad6af99ffd94e0f54e89dfc35b0fb7252ead2ed0ad2dca431bdd0'],
+    }),
+    ('click', '8.1.3', {
+        'checksums': ['7682dc8afb30297001674575ea00d1814d808d6a36af415a82bd481d37ba7b8e'],
+    }),
+    ('cligj', '0.7.2', {
+        'checksums': ['a4bc13d623356b373c2c27c53dbd9c68cae5d526270bfa71f6c6fa69669c6b27'],
+    }),
+    ('simplejson', '3.17.6', {
+        'checksums': ['cf98038d2abf63a1ada5730e91e84c642ba6c225b0198c3684151b1f80c5f8a6'],
+    }),
```

Comment:
```suggestion
```

This is in [Python/3.10.4](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.10.4-GCCcore-11.3.0.eb#L428)